app/notification_email.go

```diff
@@ -99,8 +99,8 @@ func (a *App) sendNotificationEmail(notification *postNotification, user *model.
 		subjectText = getNotificationEmailSubject(user, post, translateFunc, a.Config().TeamSettings.SiteName, team.DisplayName, useMilitaryTime)
 	}
 
-	teamURL := a.GetSiteURL() + "/" + team.Name
-	var bodyText = a.getNotificationEmailBody(user, post, channel, channelName, senderName, team.Name, teamURL, emailNotificationContentsType, useMilitaryTime, translateFunc)
+	landingURL := a.GetSiteURL() + "/vault#/" + team.Name
```

Comment:
@dmeza the url is really `/vault#/` ?

Reply:
@csduarte yes, not sure who proposed it, but it's been like that since almost the beginning of these changes. It's how it's in uChat production and how the PR back to MM master was created.